### PR TITLE
edit key bindings ref topic

### PR DESCRIPTION
**Pull request by guillermooo**

### CODE REVIEW

**Comment by guillermooo** [comment]
@FichteFoll PTAL

File: source/reference/key_bindings.rst [hunk 1]
```diff
@@ -57,10 +55,9 @@ Below you'll find valid elements in key bindings.
    of parameters to ``command``.
 
 ``context``
-   Array of contexts to selectively
-   enable the key binding.
-   All contexts must be true
-   for the key binding to trigger.
+   Array of contexts
+   the key binding is restricted to.
+   All contexts are *AND*ed together.
```
Comment (FichteFoll) [inline source/reference/key_bindings.rst]: I prefer the previous version. Maybe "all context requirements must hold (true)"?

Comment (guillermooo) [inline source/reference/key_bindings.rst]: I'll try to revert to something similar to what we had before.

**Comment by guillermooo** [comment]
@FichteFoll Tried to edit the part you pointed out. I'll have to squash these commits before merging, but PTAL.

**Comment by FichteFoll** [comment]
LGTM.

**Pull request closed without merging**